.github/workflows/main.yaml

```diff
@@ -290,6 +279,64 @@ jobs:
           if-no-files-found: error
           retention-days: 1
 
+  test-docker:
+    name: Test on ${{ matrix.build }}
+    runs-on: ubuntu-latest
+    needs: setup
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - build: linux-musl-x64
+            image: alpine:latest
+            rust: 1.49
+            artifact_name: 'wasmer-linux-musl-amd64'
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: addnab/docker-run-action@v1
+        with:
+          image: ${{ matrix.image }}
+          options: -v ${{ github.workspace }}:/work
```

Comment:
is the `:/` intentional or should this just be `/`?

Reply:
It intentional, this part is a Docker volume mapping which syntax is `<path-on-the-host>:<path-in-the-container>`